lib/tapioca/dsl/compilers/active_record_relations.rb

```diff
@@ -679,6 +680,59 @@ def create_common_methods(common_relation_methods_module)
             end
           end
 
+          BATCHES_METHODS.each do |method_name|
+            case method_name
+            when :find_each
+              create_common_method(
+                "find_each",
+                common_relation_methods_module,
+                parameters: [
+                  create_kw_opt_param("start", type: "T.untyped", default: "nil"),
```

Comment:
I'm assuming the primary keys can be other types than an Integer and so we are leaving them `T.untyped`?